src/app/templates/app/submit_new_license.html

```diff
@@ -69,6 +69,20 @@
 <form id="newlicense" enctype="multipart/form-data" class = "form-horizontal" method = "post" action='/submit_new_license/'>
 		{% csrf_token %}
 
+    <div class = "form-group">
+      <div class="col-sm-12">
+      <label class="control-label col-sm-3" >Explain why this license meets the
+        <a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-inclusion-principles.md">
+          license inclusion principles
+        </a>
+        <span class="glyphicon glyphicon-question-sign" id="commentsInfo" data-toggle="popover" data-placement="auto left"></span>
+      </label>
+        <div class="col-sm-6">
+          {{ form.comments }}
+        </div>
+      </div>
+    </div>
+
```

Comment:
Can you put a screenshot for this change? Non-dev people will also be able to review your changes without setting up the repo?